### PR TITLE
feat: add pressure

### PR DESCRIPTION
**Pull request by erikbohnsack**

### CODE REVIEW

File: pressure.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+const kiloPascalSymbol = "kPa"
+
+// KiloPascal returns p with the unit of kPa.
+func (p Pressure) KiloPascal() float64 {
+	return p.Get(KiloPascal)
+}
+
+// Bar is 100000 times the SI unit Pascal.
+const Bar Pressure = 1e6 * Pascal
```
Comment (JoakimKjellberg) [inline pressure.go:30]: The comment is correct, but the code is not. This should be `1e5`.

File: pressure_test.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package unit
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestPressure_Get(t *testing.T) {
+	assert.Equal(t, float64(3), (3000 * KiloPascal).Get(Bar))
```
Comment (JoakimKjellberg) [inline pressure_test.go:10]: And the expected value should be 30 instead of 3.

**Review by JoakimKjellberg — CHANGES_REQUESTED** [review]
Thanks for doing this! I found one conversion error.

File: pressure.go (hunk 1 above)
Comment (erikbohnsack) [reply pressure.go]: Very good catch, thanks!

**Review by JoakimKjellberg — APPROVED** [review]
Looks good! Just Emil's comment left to fix.

**Pull request merged**